### PR TITLE
add priorityClassName: rhoam-pod-priority to rhmi.cr.yaml

### PR DESCRIPTION
**Pull request by austincunningham**
# Description

add priorityClassName: rhoam-pod-priority to rhmi.cr.yaml

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] This change requires a documentation update 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added a test case that will be used to verify my changes 
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by matskiv** [comment]
LGTM, but I would like to hear @CathalOConnorRH opinion, since he knows most about the new field.

File: deploy/crds/examples/rhmi.cr.yaml [hunk 1]
```diff
@@ -10,3 +10,4 @@ spec:
   smtpSecret: redhat-rhmi-smtp
   deadMansSnitchSecret: redhat-rhmi-deadmanssnitch
   pagerDutySecret: redhat-rhmi-pagerduty
+  priorityClassName: rhoam-pod-priority
```
Comment (CathalOConnorRH) [inline deploy/crds/examples/rhmi.cr.yaml:13]: In theory this should be fine, I just wonder should we also change https://github.com/integr8ly/integreatly-operator/blob/master/pkg/controller/installation/installation_controller.go#L60 as part of this to keep the name consistent?

Comment (austincunningham) [reply deploy/crds/examples/rhmi.cr.yaml:13]: no objections from me with that change

**Comment by KevFan** [comment]
Not too sure but `managed-service-priority` is also reference here, which might also need to be updated if the name is changed (or better passed in as a variable to this function from the RHMI CR) :thinking: 
* https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/podPriority.go#L41

**Comment by matskiv** [comment]
@KevFan oh, that's interesting.
If the class is configurable via CR, it shouldn't be hard coded here: https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/podPriority.go#L41
right?

**Comment by KevFan** [comment]
> @KevFan oh, that's interesting.
> If the class is configurable via CR, it shouldn't be hard coded here: https://github.com/integr8ly/integreatly-operator/blob/master/pkg/resources/podPriority.go#L41
> right?

Yeah, that's my understanding, otherwise it's always updated to the hardcoded value :thinking: This might have been missed in the original PR or I'm misunderstanding something :confused:

**Pull request merged**